testdata/project-v4-with-grafana/test/utils/utils.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
```

Comment:
It is missing the funcs to install/uninstall cert-manager and Prometheus
See: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v4-alpha/memcached-operator/test/utils/utils.go#L29-L108